swap_meet/item.py

```diff
@@ -1,2 +1,27 @@
 class Item:
-    pass
+    #Wave 02
+    def __init__(self,category= "", condition = 0):
+        self.category = category 
+        self.condition = condition
+    
+    #Wave 03
+    def __str__(self):
+        return "Hello World!"
+
+    #Wave 05 
+    def condition_description(self ):
+        condition = self.condition
+        if condition >= 0 and condition < 1:
+            description = "yikes,raggedy, on its last leg"
+        if condition >= 1 and condition < 2:
+            description = "eek,heavy use,'rough around the edges'"
+        if condition >= 2 and condition < 3:
+            description = "eh,well used"
+        if condition >= 3 and condition < 4:
+            description = "ok,expected wear and tear"
+        if condition >= 4 and condition < 5:
+            description = "good,light use"
+        if condition == 5:
+            description = "superb,outstanding,like new"
+        return description
```

Comment:
Using ranges was a great approach and I love the descriptions. Here is a suggestion to use less lines of code
 ```suggestion
      if self.condition > 4.0:
            return "Very good condition"
        elif 3.0 < self.condition <= 4.0:
            return "Pretty good condition"
        elif  2.0 < self.condition <= 3.0:
            return "Noticeable wear and tear"
        else:
            return "Fashionably rustic"
```